pkg/server/container_create.go

```diff
@@ -121,3 +168,283 @@ func (c *criContainerdService) CreateContainer(ctx context.Context, r *runtime.C
 
 	return &runtime.CreateContainerResponse{ContainerId: id}, nil
 }
+
+func (c *criContainerdService) generateContainerSpec(id string, sandboxPid uint32, config *runtime.ContainerConfig,
```

Comment:
Makes sense that this code needed to move from start to create given the containerd changes... but it makes it hard to review :-)